assets/styles/ecli.scss

```diff
@@ -12,8 +12,6 @@ $exo3:      'exoblack', 'Helvetica Neue', Helvetica, Arial, sans-serif;
 
 /*  BASE STYLES   */
 
-.hidefirst {display: none;}
```

Comment:
It would probably be better to [remove the `hideFirst` Class from the template](https://github.com/ember-cli/ember-cli/blob/gh-pages/_includes/description.html#L3).